example/cpp/README.md

```diff
@@ -180,6 +180,9 @@ OPTIONS
        --no-value
            do not display a value for each symbol
 
+       -o FILE, --output=FILE
+           Write output to a file.
```

Comment:
Which output is this? The model? The Wasm file generated by compiling the C++ programs?

Reply:
OK, you may also change the name of the option? Removing `-o` and only having `--wasm-output` for instance?